api/client.go

```diff
@@ -149,6 +149,18 @@ func testResponseCode(expectedStatusCode int, response *http.Response) error {
 	return nil
 }
 
+func (hb HttpClient) SignJWS(payload []byte, entity types.LegalEntity) ([]byte, error) {
```

Comment:
Perhaps implementing this? Otherwise the lib cannot be used in http-client anymore which breaks running and developping seperate engines.

Reply:
I introduced the client implementation in this PR because I needed it to be able to introduce a remotely executed CLI command. There no engine depending on this client implementation yet so it's not called anywhere.

There's also functions in the `crypto` interface that in my opinion shouldn't be exposed with the level of security our internal APIs currently have. I'm thinking of functions that return private keys first and functions that sign, second. These aren't  even exposed in the REST API (except JWT signing) currently.

It's debatable whether there should be unsupported functions in the client implementation, but implementing them is comletely out of scope for this issue/PR anyways.

Reply:
Can you create an issue where this is addressed and refer to it from this PR?

Reply:
Made an issue: https://github.com/nuts-foundation/nuts-crypto/issues/75